samples/analyze/beta_snippets_test.py

```diff
@@ -176,3 +176,14 @@ def test_streaming_automl_classification(capsys, video_path):
     beta_snippets.streaming_automl_classification(video_path, project_id, model_id)
     out, _ = capsys.readouterr()
     assert "brush_hair" in out
+
+# Flaky Gateway
+@pytest.mark.flaky(max_runs=3, min_passes=1)
+def test_streaming_automl_object_tracking(capsys, video_path):
+    project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+    # TODO model_id for VOT
+    model_id = "VOT282620667826798592"
+    beta_snippets.streaming_automl_object_tracking(video_path, project_id, model_id)
+    out, _ = capsys.readouterr()
+    assert "cat" in out
```

Comment:
NIT: We may not want to check for the actual output from the model, as there's a chance that future changes to the model can change the output.